protocol/contact.go

```diff
@@ -83,6 +82,16 @@ func (c Contact) IsBlocked() bool {
 	return existsInStringSlice(c.SystemTags, contactBlocked)
 }
 
+func (c *Contact) Remove() {
+	var newSystemTags []string
+	for _, tag := range newSystemTags {
+		if tag != contactAdded {
+			newSystemTags = append(newSystemTags, tag)
+		}
+	}
+	c.SystemTags = newSystemTags
```

Comment:
I don't get what this method is doing.

Reply:
It's removing the `newSystemTag` from a contact (effectively removing the contact). I will add a comment